realsense2_camera/src/rs435_node.cpp

```diff
@@ -98,18 +98,19 @@ void RS435Node::setParam(rs435_paramsConfig &config, rs435_param param)
         _sensors[DEPTH].set_option(rs2_option::RS2_OPTION_EXPOSURE, config.rs435_depth_exposure * rs435_depth_exposure_factor);
     }
         break;
-    /*case rs435_depth_laser_power:
```

Comment:
@barabas6river Sorry I missed this comment. Basically, this is meant to avoid a warning message when starting up camera. We can only restore that when the driver can set laserpower properly and reliably